app/styles/components/_jukebox-player.scss

```diff
@@ -0,0 +1,187 @@
+$white: $white;
+$white: $white;
+
+@import '../mixins/mq';
+$nypr-player-medium-breakpoint               : #{$mq_tablet-portrait};
+$nypr-player-medium-and-up                   : "(min-width: #{$nypr-player-medium-breakpoint})" !default;
+$nypr-player-small-only                      : "(max-width: #{$nypr-player-medium-breakpoint - 1})" !default;
+
+$nypr-player-primary-color                   : $white;
+$nypr-player-background-color                : darken($denim-blue, 2%);
+$nypr-player-accent-color                    : $white;
+
+$nypr-player-button-color                    : $white;
+$nypr-player-button-hover-color              : rgba($white, 0.8);
+$nypr-player-button-active-color             : $white;
+$nypr-player-slider-handle-color             : $white;
+
+$nypr-player-info-text-color                 : $white;
+$nypr-player-link-underline-color            : $white;
+$nypr-player-timelabel-color                 : $white;
+$nypr-player-timelabel-total-color           : rgba($white, 0.6);
+
+$nypr-player-backdrop-color                  : rgba($white, 0.75);
+$nypr-player-progress-played-color           : $white;
+$nypr-player-progress-downloaded-color       : darken($denim-blue, 20%);
+
+
+.nypr-player.jukebox-player {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  height: 95px;
+  width: 100%;
+  padding-bottom: 15px;
+  position: fixed;
+  bottom: -15px;
+  z-index: 1200;
+  box-sizing: border-box;
+
+  @include media('<medium') {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    padding: 10px 15px!important;
+  }
+
+  background-color: $nypr-player-background-color!important;
+
+  //
+  // .svg-path-circle-icon.outer {
+  //   fill: transparent;
+  //   color: transparent;
+  // }
+  //
+  // .nypr-player-button.mod-listen .svg-path-circle-icon.outer {
+  //   stroke: $global-player-button-stroke;
+  // }
+  //
+  // .nypr-player-button.mod-listen:active .svg-path-circle-icon.outer {
+  //   stroke: $white;
+  // }
+  //
+  // .nypr-player-button.mod-rewind path:first-child,
+  // .nypr-player-button.mod-fastforward path:first-child{
+  //   fill: $global-player-button-stroke;
+  // }
+  //
+  // .nypr-player-button.mod-rewind:active path:first-child,
+  // .nypr-player-button.mod-fastforward:active path:first-child{
+  //   fill: $white;
+  // }
+}
```

Comment:
Ack, I left this commented out CSS because I still need to try and reduce the stroke-width of that button but stalled on it.